main/php_network.h

```diff
@@ -73,9 +73,13 @@ END_EXTERN_C()
 #include <sys/socket.h>
 #endif
 
-#ifdef HAVE_GETHOSTBYNAME_R
+#ifdef HAVE_GETADDRINFO
+#ifdef PHP_WIN32
+#include "win32/ws2tcpip.h" #TODO: does this work?
```

Comment:
It should probably be just `#include <Ws2tcpip.h>`. (We already have a couple usages of this header.)